src/unity/toolkits/coreml_export/MLModel/build/format/CategoricalMapping_enums.h

```diff
@@ -8,6 +8,7 @@ enum MLCategoricalMappingMappingType: int {
     MLCategoricalMappingMappingType_NOT_SET = 0,
 };
 
+__attribute__((__unused__))
```

Comment:
This is ultimately generated code, right? Will we have to make these modifications each time we update our snapshot of the proto?

Reply:
Yes. I plan to put up a coremltools PR to fix it in the generator.

Reply:
Fixed on the coremltools side: https://github.com/apple/coremltools/pull/327